test_framework/Jenkinsfile

```diff
@@ -136,6 +136,7 @@ node {
                                        --env TF_VAR_azure_tenant_id=${AZURE_TENANT_ID} \
                                        --env TF_VAR_azure_subscription_id=${AZURE_SUBSCRIPTION_ID} \
                                        --env TF_VAR_cis_hardening=${CIS_HARDENING} \
+                                       --env TF_VAR_custom_ssh_public_key="${CUSTOM_SSH_PUBLIC_KEY}" \
```

Comment:
Would this be exported in the log of a job build? If yes, we should use this only for authorized users.

Reply:
@innobead This PR https://github.com/longhorn/infra/pull/151 isn't merged yet, which only provides `CUSTOM_SSH_PUBLIC_KEY` option in `private/longhorn-tests-regression` job.